src/sentry/incidents/endpoints/organization_alert_rule_index.py

```diff
@@ -37,14 +46,53 @@ def get(self, request, organization):
                 "id", flat=True
             )
 
+        teams = request.GET.getlist("team", [])
+        team_filter_query = None
+        if teams:
+            unassigned = None
+            if "unassigned" in teams:
+                teams.remove("unassigned")
+                unassigned = Q(owner_id=None)
+
+            if "myteams" in teams:
+                teams.remove("myteams")
+                for t in Team.objects.filter(
+                    id__in=OrganizationMemberTeam.objects.filter(
+                        organizationmember__in=OrganizationMember.objects.filter(
+                            user=request.user, organization_id=organization.id
+                        ),
+                        is_active=True,
+                    ).values("team")
+                ):
+                    teams.append(t.id)
```

Comment:
I think we can just do `request.teams` here, it should use the same query.

Reply:
Ah you're right. They're the same in every test. Didn't realize we had this.